tests/engine/render/shadows/testRecursive.js

```diff
@@ -1,12 +1,16 @@
-// This is an automatically generated file, created by auto-tester on May 16 2018, 9:09
+// This is an automatically generated file, created by auto-tester on May 24 2018, 13:34
+
+user = "highfidelity/"
+repository = "hifi_tests/"
+branch = "master/"
```

Comment:
The semicolons at the end of JS lines are optional, right? But they are preferred?